asv_bench/benchmarks/benchmarks_perfect_model.py

```diff
@@ -2,17 +2,28 @@
 # See "Writing benchmarks" in the asv docs for more information.
 
 
-import xarray as xr
+import dask
 import numpy as np
-from . import randn, parameterized
+import xarray as xr
 
-from climpred.prediction import compute_perfect_model
 from climpred.bootstrap import bootstrap_perfect_model
-from climpred.constants import PM_COMPARISONS, PM_METRICS
+from climpred.prediction import compute_perfect_model
+
+from . import parameterized, randn, requires_dask
+
+# faster than
+# from climpred.constants import PM_COMPARISONS, PM_METRICS as METRICS
```

Comment:
This can also be updated

Reply:
I only want to take a subselection here. this applies to the two asv files. comment added